### PR TITLE
Add new tokens for `ViewSwitcher`

### PR DESCRIPTION
**Pull request by Rhotimee**
#### Summary
Adjust ViewSwitcher component for new theme

### CODE REVIEW

File: design-system/materials/internals/definition.yaml [hunk 1]
```diff
@@ -809,6 +841,10 @@ decisionGroupsByTheme:
           choice: '0 var(--spacing-10)'
         padding-for-tooltip:
           choice: 'var(--spacing-10) var(--spacing-20)'
+        padding-for-view-switcher:
+          choice: '0 var(--spacing-30) 0 var(--spacing-30)'
+        padding-for-view-switcher-when-condensed:
+          choice: '0 var(--spacing-20) 0 var(--spacing-20)'
```
Comment (CarlosCortizasCT) [inline design-system/materials/internals/definition.yaml:847]: Horizontal padding for each button when condensed should be 12px according to the [Miro board](https://miro.com/app/board/uXjVPSlrfYY=/?moveToWidget=3458764542513986030&cot=14) but currently is 8px.

File: design-system/materials/internals/definition.yaml [hunk 2]
```diff
@@ -996,6 +1042,10 @@ decisionGroupsByTheme:
       decisions:
         box-shadow-for-datetime-input-when-hovered:
           choice: 'inset 0 0 0 1px'
+        box-shadow-for-view-switcher:
+          choice: 'none'
+        box-shadow-for-view-switcher-when-selected:
+          choice: 'none'
```
Comment (CarlosCortizasCT) [inline design-system/materials/internals/definition.yaml:1048]: **nit**: your could use `shadow-0` choice token instead of `none`.

**Review by CarlosCortizasCT — COMMENTED** [review]
Looks great 👏 

Just left some padding issue I found.

File: design-system/materials/internals/definition.yaml (hunk 1 above)
Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml:847]: Nice catch. I actually added the token but forgot to update the component. 
Heres the commit -> [2688827](https://github.com/commercetools/ui-kit/pull/2390/commits/2688827749ddfa673a5793503cc7145f164531ed)
Thanks.

Comment (CarlosCortizasCT) [reply design-system/materials/internals/definition.yaml:847]: But I mean, `spacing-20` is a token with the value `8px` and in the Miro board it says we should be using `12px` for horizontal spacing when condensed.

Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml:847]: This choice is for the default theme though. The default theme uses 8px for padding when condensed.

Comment (CarlosCortizasCT) [reply design-system/materials/internals/definition.yaml:847]: My bad. Sorry 🤦

**Review by CarlosCortizasCT — APPROVED** [review]
👏

**Review by ddouglasz — APPROVED** [review]
Nice work 🙌🏾

**Review by FilPob — APPROVED** [review]

**Pull request merged**